src/screens/HomePage/HomePage.style.tsx

```diff
@@ -18,16 +18,41 @@ export const Content = styled.div`
   margin: auto auto 3rem;
 `;
 
-export const Section = styled.div`
+export const HomePageBlock = styled.div`
   margin: 3.5rem 1rem;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    margin: 2.75rem 1rem 4.75rem;
```

Comment:
Desktop has slightly less space above the vaccinations map block and slightly more space between each block. 🤷